scripts/generate-coverage.sh

```diff
@@ -6,7 +6,7 @@
 set -e
 echo "" > coverage.txt
 go test -i -race ./cmd/odo
-for d in $(go list ./... | grep -v vendor | grep -v tests/e2e); do
+for d in $(go list ./... | grep -v vendor | grep -v tests/e2e | grep -v testingutil); do
```

Comment:
Sneaky! :)

Reply:
Unfortunately it didn't have the effect I had hoped for :)